third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -5,9 +5,34 @@
 //! Implements a live-variable analysis processor, annotating lifetime information about locals.
 //! See also https://en.wikipedia.org/wiki/Live-variable_analysis
 //!
+//! prerequisite annotations: none
+//! side effect: the `LiveVarAnnotation` will be added to the function target annotations.
+//!
 //! This processor assumes that the CFG of the code has no critical edges.
+//!
+//! Notes on shadowed use (this terminology is used in this module):
```

Comment:
Using the 'shadow' notion here in context with variables is a bit confusing, as shadowing already has a meaning for variables. Perhaps primary vs secondary use? But actually not sure what the purpose of this when reading just this text.

Reply:
Done, based on our discussion.